src/components/Table/BaseTable.tsx

```diff
@@ -131,12 +132,13 @@ export const BaseTable = <T extends LGRowData>({
             `}
           >
             {row.getVisibleCells().map((cell) => (
-              <Cell key={cell.id}>
+              <Cell key={cell.id} style={{ padding: `${size.xxs} 2px` }}>
```

Comment:
Can we use emotion to style this entirely? Also what is this style for?

You should be able to set the displayName property on the styled component if LG expects a Cell.

Reply:
We can't as a result of [this bug](https://jira.mongodb.org/browse/LG-3861) =( there is also a problem with applying styles to cells have have expandable content.

This sets some padding within each cell so that the cell contents aren't directly touching.